### PR TITLE
ir: Always enable RPC sessions in local consensus launch mode

### PR DESCRIPTION
**Pull request by cthulhu-rider**
@roman-khimov @AnnaShaleva note that i've left other session-related configurations unset https://pkg.go.dev/github.com/nspcc-dev/neo-go@v0.101.4/pkg/config#RPC, if u have any suggestions about non-default values pls let me know

* see also https://github.com/nspcc-dev/neo-go/issues/3106

### CODE REVIEW

File: pkg/innerring/internal/blockchain/blockchain.go [hunk 1]
```diff
@@ -370,6 +370,9 @@ func New(cfg Config) (res *Blockchain, err error) {
 	cfgBaseApp.P2PNotary.UnlockWallet = cfg.Wallet
 	cfgBaseApp.RPC.StartWhenSynchronized = true
 	cfgBaseApp.RPC.MaxGasInvoke = fixedn.Fixed8FromInt64(100)
+	cfgBaseApp.RPC.SessionEnabled = true
+	// MaxIteratorResultItems must be explicitly set for rpcsrv.New because it doesn't set default
```
Comment (AnnaShaleva) [inline pkg/innerring/internal/blockchain/blockchain.go:374]: I think we may safely remove this comment, https://github.com/nspcc-dev/neo-go/pull/3107 will be included into 0.102.0

**Review by AnnaShaleva — COMMENTED** [review]
Otherwise LGTM.

File: pkg/innerring/internal/blockchain/blockchain.go (hunk 1 above)
Comment (cthulhu-rider) [reply pkg/innerring/internal/blockchain/blockchain.go:374]: the comment explains why we set default manually (only this and no one else), think it's useful. With next release both comment and instruction will disappear

**Review by roman-khimov — APPROVED** [review]

**Review by carpawell — APPROVED** [review]
CHANGELOG can be changed.

**Pull request merged**